strapi/strapi
# Make select autocomplete case insensitive

### What does it do?

Solves #13888.

### Why is it needed?
The select component's autocomplete feature is currently case sensitive, meaning some results may not show up in the list if the casing does not match exactly.

### How to test it?

Create some entries with titles in different casing (eg. title case vs lowercase) and try scrolling to the bottom of the list.
Notice how the component also shows entries with titles in different casing, whereas the component in the current Strapi release doesn't.

### Related issue(s)/PR(s)

#13888.
